packages/plugin-core/src/test/suite-integ/CapitalizeText.test.ts

```diff
@@ -0,0 +1,123 @@
+import { ENGINE_HOOKS } from "@dendronhq/common-test-utils";
+import _ from "lodash";
+import { ExtensionContext, Position, Selection } from "vscode";
+import { CapitalizeTextCommand } from "../../commands/CapitalizeText";
+import { VSCodeUtils } from "../../utils";
+import { expect } from "../testUtilsv2";
+import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
+
+suite("notes", function () {
+  let ctx: ExtensionContext;
+
+  ctx = setupBeforeAfter(this, {});
+
+  test("with no selections", (done) => {
+    runLegacyMultiWorkspaceTest({
+      ctx,
+      preSetupHook: ENGINE_HOOKS.setupBasic,
+      onInit: async ({ vaults }) => {
+        const vault = vaults[0];
+        const fname = "foo";
+
+        const editor = await VSCodeUtils.openNoteByPath({ vault, fname });
+
+        const lowercaseText = "undercase text";
+
+        const start = new Position(8, 0);
+        const end = new Position(8, lowercaseText.length);
+        const selection = new Selection(start, end);
+
+        await editor.edit((editBuilder) => {
+          editBuilder.insert(start, lowercaseText);
+        });
+
+        await new CapitalizeTextCommand().run();
+
+        const changedText = editor.document.getText(selection);
```

Comment:
this method name is confusing since the text isn't actually changed

Reply:
Would the name `CapitalizeSelectedTextCommand` be clearer?